src/service.js

```diff
@@ -260,6 +285,16 @@ class Service {
 		return action;
 	}
 
+	/**
+	 * Retrieve list of active contexts for the service
+	 * @returns {Set}
+	 * @private
+	 * @memberof Service
+	 */
+	_getActiveContexts() {
+		return this.activeContexts || [];
```

Comment:
Please use `_` before local variable name because it can conflict with user's service variables. Use as `this._activeContexts`